playbooks/ContinuousDelivery/image_promotion.adoc

```diff
@@ -0,0 +1,117 @@
+= OpenShift Enterprise 3 Image Promotion Strategies
+Andrew Block <ablock@redhat.com>
+v1.0, 2015-10-05
+:scripts_repo: https://github.com/rhtconsulting/rhc-ose
+:toc: macro
+:toc-title:
+
+toc::[]
+
+This document provides strategies for promoting an application between various OpenShift environments. The level of segregation of how environments are divided are at the discretion of the user. It is out of the scope of this particular document on how the application is built and packed into the initial image
+
+== High Level Components
+
+Each strategy defined within this document will have several consistent themes:
+ 
+* 1:x OpenShift Environments
+* Integration environment
+** It is _recommended_ that the Jenkins CI server be used to facilitate the promotion process, however standalone scripts may be used instead if desired
+Docker client to manipulate images
+The user executing the scripts must be a member of the docker group in order to access the Docker socket. Additional information on this topic can be found https://docs.docker.com/installation/fedora/[here]
```

Comment:
The entire section? Overview? or high level components should be in an a Admonition block?